src/common/cast/remote-player.js

```diff
@@ -121,21 +121,6 @@ export interface IRemotePlayer {
    * @memberof IRemotePlayer
    */
   +config: Object;
-  /**
-   * @readonly
-   * @static
-   * @type {string}
-   * @memberof IRemotePlayer
-   */
-  static +Type: string;
-
-  /**
-   * @static
-   * @method
-   * @returns {boolean}
-   * @memberof IRemotePlayer
-   */
-  static isSupported(): boolean;
```

Comment:
@dan-ziv why remove? because of static? if this is required then define IRemotePlayerStatics and add it there.